Helm lint throws errors on chart.yaml

Helm lint command fails on helm files created due to following issues:
- [ERROR] Chart.yaml: directory name (XXX_DIR) and chart name (A new XXX project) must be the same
- [ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"

We could fix these issues and contribute to the project. Please let us know if we could create a PR for this.

--- a/pkg/transformer/kubernetes/k8sutils.go
+++ b/pkg/transformer/kubernetes/k8sutils.go
@@ -88,6 +88,7 @@ func generateHelm(dirName string) error {
 	chart := `name: {{.Name}}
 description: A generated Helm Chart for {{.Name}} from Skippbox Kompose
 version: 0.0.1
+apiVersion: v1
 keywords:
   - {{.Name}}
 sources:

